tests/scripts/generate_test_code.py

```diff
@@ -519,6 +519,41 @@ def generate_function_code(name, code, local_vars, args_dispatch,
         gen_dependencies(dependencies)
     return preprocessor_check_start + code + preprocessor_check_end
 
+COMMENT_START_REGEX = re.compile(r'/[*/]')
+
+def skip_comments(line, stream):
+    """Remove comments in line.
+
+    If the line contains an unfinished comment, read more lines from stream
+    until the line that contains the comment.
+
+    :return: The original line with inner comments replaced by spaces.
```

Comment:
So that column positions remain correct in error messages. (And arguably line breaks inside comments should stay as line breaks, but I've filed that under “if someone else cares enough they'll work on it”.)

(What column positions, you ask? Well, maybe someone will improve error messages one day.)